test/static_assertions/CMakeLists.txt

```diff
@@ -41,7 +41,7 @@ if (UNIX)
     PASS_REGULAR_EXPRESSION "is not a base class of")
 
   set_tests_properties(STATIC_ASSERT_plugin_not_global_ns PROPERTIES
-    PASS_REGULAR_EXPRESSION "IGN_macro_must_be_used_in_global_namespace")
+    PASS_REGULAR_EXPRESSION "GZ_macro_must_be_used_in_global_namespace")
```

Comment:
I think this is causing test failures:

```
  [100%] Building CXX object test/static_assertions/CMakeFiles/plugin_not_global_ns.dir/plugin_not_global_ns.cc.o
  In file included from /github/workspace/include/ignition/common/PluginMacros.hh:23,
                   from /github/workspace/test/static_assertions/plugin_not_global_ns.cc:22:
  /github/workspace/include/ignition/common/detail/PluginMacros.hh:71:9: [Censored by testrunner.bash]: '::IGN_macro_must_be_used_in_global_namespace' has not been declared; did you mean 'ns::IGN_macro_must_be_used_in_global_namespace'?
     71 |       ::IGN_macro_must_be_used_in_global_namespace>::value, \
        |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
Fixed! https://github.com/gazebosim/gz-common/pull/392/commits/92a820c4c698336c68a3dee81d0fa9af9eb444aa